models/knex/knexfile.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const config = require('../../config');
+
+const { postgresql, dataService } = config;
+
+const { host, user, password, database } = postgresql;
+const { model } = dataService;
+
+module.exports = {
+  development: {
+    client: `${model}`,
```

Comment:
What's the benefit of coercing this to string instead of, e.g.:
```suggestion
    client: model,
```